### PR TITLE
add malaysian non iso currency RM => MYR

### PR DESCRIPTION
**Pull request by Amos47**
## Problem
RM isn't recognized as a currency code.

## Solution
Add a mapping to the MYR ISO code.

@Smcchoi  @KnVerey 

@girasquid

### CODE REVIEW

File: lib/active_utils/currency_code.rb [hunk 1]
```diff
@@ -29,6 +29,7 @@ class CurrencyCode
       "NMP" => "MXN", # Mexican Pesos
       "NTD" => "TWD", # New Taiwan Dollars / Taiwan New Dollars
       "RDD" => "DOP", # Dominican Pesos
+      "RB" => "MYR",  # Malaysia Ringgit
```
Comment (girasquid) [inline lib/active_utils/currency_code.rb]: Description says RM but this is RB - which is it?

Comment (Amos47) [inline lib/active_utils/currency_code.rb]: RM.. Changed now.

**Comment by girasquid** [comment]
:+1:

**Comment by KnVerey** [comment]
:+1:

**Pull request merged**